core/src/main/scala/org/apache/spark/ExecutorAllocationManager.scala

```diff
@@ -292,9 +294,8 @@ private[spark] class ExecutorAllocationManager(
   private def addExecutors(maxNumExecutorsNeeded: Int): Int = {
     // Do not request more executors if it would put our target over the upper bound
     if (numExecutorsTarget >= maxNumExecutors) {
-      val numExecutorsPending = numExecutorsTarget - executorIds.size
-      logDebug(s"Not adding executors because there are already ${executorIds.size} registered " +
-        s"and ${numExecutorsPending} pending executor(s) (limit $maxNumExecutors)")
+      logInfo(s"Not adding executors because our current target total " +
```

Comment:
What's the rationale behind bumping this up to info? It seems like there will be common situations (any time that there are sustained pending tasks but we've hit our max) where this will end up printing a log every second.  Which is a lot of noise.